services/src/handlers/wcs.rs

```diff
@@ -440,7 +440,11 @@ async fn wcs_get_coverage_handler<C: Context>(
         .await)?
     .map_err(error::Error::from)?;
 
-    Ok(HttpResponse::Ok().content_type("image/tiff").body(bytes))
+    assert_eq!(bytes.len(), 1);
```

Comment:
I think we should not crash the program here. Maybe just return an error instead.

Alternatively, replace it with a debug_assert :shrug:

Reply:
I added a convenience function for single timesteps and a InvalidNumberOfTimeSteps error if it is not a single timestep. Removes redundant code in tests and allows a map to service error here, so I feel such a convenience function is justified. 8b7769d2a5ca85f0dd06a8268905e3a3f94c19cf